Document loading records from submitted forms (#7381)
umbraco/UmbracoDocs: 16/umbraco-forms/developer/working-with-data.md

    }
</ul>
```

## Loading a Record From a Submitted Form

> When a form is submitted, the submitted form id and the saved record id is stored in the TempData so it can be referenced.
> 
> You can use the FormService and the RecordStorage to get the ```Form``` and ```Record``` objects.   
> 
> Here is sample code for how to get the record in a view.

Reviewer: ```suggestion
When a form is submitted, the submitted form ID and the saved record ID are stored in the `TempData` so they can be referenced.

You can use the FormService and the RecordStorage to get the `Form` and `Record` objects.

Here is a sample code for retrieving a record in a view.
```

```
@using Umbraco.Forms.Core.Models
@using Umbraco.Forms.Core.Persistence.Dtos
@using Umbraco.Forms.Core.Data.Storage
@using Umbraco.Forms.Core.Services
@inject IFormService _formService
@inject IRecordStorage _recordStorage
@inherits UmbracoViewPage
@{
	Guid formId;
	Form? form;
	Guid recordId;
	Record? record;
    string submittedEmail;

> 	if (TempData["UmbracoFormSubmitted"] != null)
> 	{
> 		Guid.TryParse(TempData["UmbracoFormSubmitted"]?.ToString(), out formId);

Reviewer: ```suggestion
	if (Guid.TryParse(TempData["UmbracoFormSubmitted"]?.ToString(), out Guid formId) &&
	    Guid.TryParse(TempData["Forms_Current_Record_id"]?.ToString(), out Guid recordId))
	{
```
